### ISSUE TITLE
streaming inference and diarization

### ISSUE THREAD

**Issue by we11as22** [salute-developers/GigaAM#18]
Are you planing to add streaming inference and diarization support?

**Comment by iasolis** [comment]
Yes, it’s also interesting to know about plans for diarization models

**Comment by xsa-dev** [comment]
And what about streaming? Do you plan to add streaming inference?

**Comment by georgygospodinov** [comment]
Thank you for your interest!

We are indeed considering adding support for punctuation/capitalization, streaming inference, and diarization in future releases. Your feedback helps us prioritize new features, so we truly appreciate it!

Stay tuned for updates, and please let us know if you have any other suggestions or requirements.

**Comment by mmarsreg** [comment]
It recognizes words quite well and quickly, but it really lacks punctuation, especially when you need to recognize a question phrase. 

When do you plan to release the update?

**Comment by ivkond** [comment]
@georgygospodinov, is there any documentation available on how to apply speaker diarization when using the new model?

**Comment by slavonnet** [comment]
#55
